proto/sentium/api/v1/relations.proto

```diff
@@ -105,13 +106,27 @@ message RelationsCreateRequest {
 
 	optional google.protobuf.Struct attrs = 7;
 
-	optional bool   optimise   = 8;
+	// Indicates whether to optimise for lookups by computing and storing derived relations.
+	// Defaults to `false`.
+	optional bool optimise = 8;
+
+	// Limits the cost for computing and storing derived relations. The value must be within `1` and `65535`.
```

Comment:
why `65535`?

Reply:
It's max `uint16_t`, which is what's used for the limit when listing tuples from the DB (e.g. [here](https://github.com/uatuko/sentium/blob/7a8daa02e241b2ae6de5a61ac564e27570be2eef/src/db/tuples.h#L115)). Protobuf doesn't have a `uint16_t` hence the explicit comment here. 

Also, don't think max `uint32_t` is a good value for DB limits.